ui/StatusQ/src/StatusQ/Popups/Dialog/+qt6/StatusFileDialog.qml

```diff
@@ -0,0 +1,37 @@
+import QtQuick 2.15
+import QtQuick.Dialogs
+
+Item {
```

Comment:
This should imho not be a (visual) `Item`; in both versions, this is a `QtObject`.